### PR TITLE
Add proto for test result predicate

### PR DESCRIPTION
**Pull request by adityasaky**

### CODE REVIEW

File: spec/predicates/test-results.md
Comment (marcelamelara) [inline spec/predicates/test-results.md:1]: Nit: might want to change the name of this file to match the proto.

**Review by marcelamelara — APPROVED** [review]
LGTM, thanks @adityasaky ! Only one small nit. I assume you'll be pushing validator APIs and/or test cases in a future PR?

File: spec/predicates/test-results.md
Comment (adityasaky) [reply spec/predicates/test-results.md:1]: Done.

**Comment by adityasaky** [comment]
> I assume you'll be pushing validator APIs and/or test cases in a future PR?

yep!

**Review by marcelamelara — APPROVED** [review]
LGTM, thanks!

**Comment by adityasaky** [comment]
@pxp928 @TomHennen @joshuagl @mikhailswift can I get a second review on this? Thanks!

**Review by pxp928 — APPROVED** [review]
LGTM! Question, is the code generation going to be done by the github workflow?

**Comment by adityasaky** [comment]
Yes, that's the idea. :D

**Pull request merged**
